nsfw/nsfw.php

```diff
@@ -95,8 +97,7 @@ function nsfw_addon_settings(&$a, &$s)
 	return;
 }
 
-function nsfw_addon_settings_post(&$a, &$b)
-{
+function nsfw_addon_settings_post(App $a, array &$b) {
```

Comment:
Code standards: Please leave the line feed before function opening brace.

Reply:
Many parts of Friendica are with curly brace at end (which I prefer) but addon with a new line.

Reply:
I understand you prefer it, but the only exception to the PSR-2 code standards we agreed on is using tabs instead of spaces for indentation. All the other rules have to be followed, including the new line before function opening brace.

Reply:
How about empty lines before closing curly brace in a function?

````
function foo ()
{
    // Do something

}
````

Reply:
@MrPetovan there is a PSR validator for checking such things. Then you have a good overview what is still left to do.

Reply:
I'm pretty much a PSR-2 validator myself at this point. And I don't like empty lines before closing braces.